django/db/migrations/recorder.py

```diff
@@ -58,8 +64,11 @@ def applied_migrations(self):
         """
         Returns a set of (app, name) of applied migrations.
         """
-        self.ensure_schema()
-        return set(tuple(x) for x in self.migration_qs.values_list("app", "name"))
+        if self.has_table():
+            return set(tuple(x) for x in self.migration_qs.values_list("app", "name"))
```

Comment:
While here, might as well switch this to use single quotes per our coding style.